step1-clean/db.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine, Column, String, Integer
+
+Base = declarative_base()
+
+class SQLImage(Base):
```

Comment:
It does not make sense to put SQL in the name here

Reply:
Read this: https://www.python.org/dev/peps/pep-0020/
`Namespaces are one honking great idea -- let's do more of those!`

This means you can do something like this:

```python
import models
image = models.Image(filename="lolcats.py")
```

Reply:
I agree. SQL was added as an ad hoc because of the PIL.Image. 

I've tried  `import PIL` then use `PIL.Image` but kept getting an error `AttributeError: module 'PIL' has no attribute 'Image'`.

However, when I use `from PIL import Image` it seems to work fine. I guess there is something I am misunderstanding, but not sure what exactly. 

To address the issue I've used `from PIL import Image as pillowImage`. Not sure if this is acceptable.

Reply:
`pillowImage` is not acceptable because class names should start with capital letters.

See the following for why `PIL.Image` didn't work:
![test](https://user-images.githubusercontent.com/1236436/77184909-e2c68d80-6aa6-11ea-8271-132b57bc8a11.png)

See [Source Code](https://github.com/python-pillow/Pillow/blob/1cecf08d16509c20473766b4cdb7a65169844819/src/PIL/Image.py#L485)

`class Image` is in file (submodule) `Image.py` in folder (module) `PIL` so that's why it's `PIL.Image.Image`

Reply:
this is a sort of rule of thumb for me. if you have name collisions in the same code, it usually means the code is not organized well and the functions are named poorly

Reply:
Okay I understand better now. From my understanding I need the `.open()`  function that is in `Image.py`, therefore `import PIL.Image` allows me to access `.open()`  from the submodule `Image.py`, which fixes my import issue. 

For further clarification, I'm assuming by using this `PIL.Image.open()` which is a factory function, using the `class Image` is not necessary,  according to the source code. So I do not need to access the `PIL.Image.Image` for `class Image` directly.

Reply:
I'll keep that rule of thumb in mind.

Reply:
The factory function is implemented as a method (function in a class) so you still need to import the class `Image` from submodule `Image.py` in module `PIL`.

Reply:
I got it working, but still think I am missing something again.

I tried to use `from PIL.Image import Image` to get the class `Image` but I get an error. It appears that there is no open (`Image.open()`)  on the class `Image` (red section below).

I tried to to look at what was available on the `Image` class (orange section below), but didn't see `.open()`. 

When I use `PIL.Image.open()` seems to work fine (blue section below).

![Screen Shot 2020-03-26 at 5 01 17 PM](https://user-images.githubusercontent.com/13709074/77699929-a523a200-6f89-11ea-90c2-909a29eb43e8.png)


Also tried it in my script. `Image.open()` did not work, but using `PIL.Image.Open()` it works. 

![Screen Shot 2020-03-26 at 4 52 10 PM](https://user-images.githubusercontent.com/13709074/77702964-10707280-6f90-11ea-8122-7a404f2ae8d9.png)